### PR TITLE
K8S-443: Make Octavia install use image in artifact storage

### PR DESCRIPTION
**Pull request by xgerman**
Adds some logic to download images, compare md5 with glance, and
update as needed.

### CODE REVIEW

**Comment by xgerman** [comment]
recheck CIT_xenial_functional_test

**Comment by xgerman** [comment]
somehow jenkins finished with success but didn't report...

File: playbooks/main.yml [hunk 1]
```diff
@@ -43,6 +43,11 @@
       tags:
         - octavia-setup
 
+    - include: rpc-octavia-install-amp-image.yml
```
Comment (jrperritt) [inline playbooks/main.yml]: Can this be delegated to `utility_all[0]`? I think there were questions raised about installing `python-openstack` client on `infra1` (around `pip` package clashes). To that end, `rpc-mk8s` now does all its OpenStack setup calls in `utility_all[0]`.

File: playbooks/vars/main.yml [hunk 2]
```diff
@@ -22,6 +22,10 @@ cert_validity_days: 1825 # 5 years
 cert_client_ca_subject: '/C=US/ST=Denial/L=Nowhere/O=Dis/CN=www.example.com' # change this to something more real
 cert_client_req_subject: '/C=US/ST=Denial/L=Nowhere/O=Dis/CN=www.example.com' # change this to something more real
 generate_client_cert: True # generate self signed client certs
+download_artefact: True
```
Comment (jrperritt) [inline playbooks/vars/main.yml]: :+1:  useful for mk8s bundler

**Review by jrperritt — COMMENTED** [review]
Just have one question around where we're installing `python-openstack`

**Review by jrperritt — APPROVED** [review]

**Pull request merged**